charts/my-bloody-jenkins/templates/_helpers.tpl

```diff
@@ -61,3 +61,7 @@ Define default values
 {{- define "my-bloody-jenkins.persistentVolumeClaimName" -}}
 {{- .Values.persistenceExistingClaim | default (include "my-bloody-jenkins.fullname" .) -}}
 {{- end -}}
+
+{{- define "my-bloody-jenkins.tlsSecretName" -}}
+{{- printf "%s-tls-secret" (include "my-bloody-jenkins.fullname" .) -}}
+{{- end -}}
```

Comment:
Please add a newline here.

Reply:
Not sure I understand what you mean. Add a new line where?

Reply:
Newlines should be added to the end of text files.
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Cool. wasn't aware of that. thanks